app/code/Magento/Theme/view/frontend/templates/html/bugreport.phtml

```diff
@@ -3,11 +3,15 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+
+/**
+ * @var \Magento\Framework\Escaper $escaper
+ */
 ?>
 <small class="bugs">
-    <span><?= $block->escapeHtml(__('Help Us Keep Magento Healthy')) ?></span>
+    <span><?= $escaper->escapeHtml(__('Help Us Keep Magento Healthy')) ?></span>
     <a href="http://www.magentocommerce.com/bug-tracking"
```

Comment:
Maybe 
```suggestion
    <a href="https://www.magentocommerce.com/bug-tracking"
```

Reply:
Should i use github link issues with https instead ? https://www.magentocommerce.com seem weird and a bit confusing
We can use https://github.com/magento/magento2/issues

Reply:
@sidolov @sivaschenko, can we put here a link to the GitHub repository?

Reply:
@mrtuvn , @ihor-sviziev yes, we can use link to GitHub